Library/Homebrew/search.rb

```diff
@@ -13,7 +13,7 @@ def query_regexp(query)
     end
 
     def search_taps(query, silent: false)
-      return [], [] if ENV["HOMEBREW_NO_GITHUB_API"]
+      return { formulae: [], casks: [] } if ENV["HOMEBREW_NO_GITHUB_API"]
```

Comment:
I think it would read a little cleaner to create this hash first thing in the function and return that. It then means `each` can be used below instead of `each_with_object`

Reply:
I don't know, in the `rescue` block `{ formulae: [], casks: [] }` feels cleaner because it is more explicit than reusing a variable that is initialized at the very beginning.

Reply:
It feels cleaner (and more DRY) to me to only have to define the `formulae` and `casks` keys in a single location.